c-c++.md

```diff
@@ -2,24 +2,28 @@
 
 ### Enabling Arm Architecture Specific Features
 
-To build code with the optimal processor features use the following.
-We recommend using Graviton2 processor features when targeting both Graviton2
-and Graviton3(E) as code compiled for Graviton3(E) will only run on Graviton3(E) and not
-on Graviton2.  On arm64 `-mcpu=` acts as both specifying the appropriate
-architecture and tuning and it's generally better to use that vs `-march` if
-you're building for a specific CPU.
+C and C++ code can be built for Graviton with a variety of flags, depending on
+the goal. If the goal is to get the best performance for a specific generation,
+select a flag from the table column "performance". If the goal is to get a good
+compromise of feature availability and performance balanced across generations,
+select the flag from the "balanced" column. We recommend using oldest generation
```

Comment:
"using" is a little ambiguous here. Do you mean compiling on?  Do you mean running on?  I think you mean "compiling _for_" which might be better said as "targeting".

Reply:
I'll reword it a little.